packages/labs/test-projects/test-elements-react/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
```

Comment:
No action comment. I wonder if in the future we could consider creating a couple base tsconfigs which are extended. Then we'll have less tsconfig duplication.

Reply:
Agree. That would make it easy for all of our packages to be consistent, and have one point for updating them all. Something to consider in the future.